python/mxnet/symbol.py

```diff
@@ -1152,6 +1153,11 @@ def var(name, attr=None, shape=None, lr_mult=None, wd_mult=None, dtype=None, ini
         attr['__dtype__'] = str(_DTYPE_NP_TO_MX[_numpy.dtype(dtype).type])
     if init is not None:
         attr['__init__'] = init.dumps()
+    for k, v in kwargs.items():
+        if k.startswith('__') and k.endswith('__'):
+            attr[k] = str(v)
+        else:
+            raise ValueError('attribute name=%s is not supported' %k)
```

Comment:
be clear, something like: "additional attributes must start and end with double underscores: __yourattr__"